src/test/java/org/folio/rtac/utils/Utils.java

```diff
@@ -1,24 +1,21 @@
-/**
- *
- */
 package org.folio.rtac.utils;
 
 import java.io.IOException;
 import java.io.InputStream;
 import java.net.ServerSocket;
 import java.util.Random;
-
 import org.apache.commons.io.IOUtils;
-
-import io.vertx.core.logging.Logger;
-import io.vertx.core.logging.LoggerFactory;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
```

Comment:
We probably need to agree on which logging framework the core modules should use too

Reply:
Agreed. I may have been a bit too bold in changing around all this stuff in this module.

Reply:
Someone has to experiment, otherwise there is no feedback to generate the change in policy

Reply:
I’ve raised a draft [issue](https://issues.folio.org/browse/FOLIO-2126)